src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -58,15 +67,20 @@
         }) }}
       {% endblock %}
 
-      {% block product_tab_stock %}
-        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/stock.html.twig', {
-          'productForm': productForm,
-        }) }}
-      {% endblock %}
-
-      {% block product_tab_combinations %}
-        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
-      {% endblock %}
+      {% if productType == 'combinations' %}
+        {% block product_tab_combinations %}
+          {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
+        {% endblock %}
+      {% else %}
+        {% block product_tab_stock %}
```

Comment:
One last thing: this whole block should be in `PrestaShop/Admin/Sell/Catalog/Product/Tabs/stock.html.twig`